builtins/amp-img.js

```diff
@@ -26,6 +27,15 @@ import {srcsetFromElement, srcsetFromSrc} from '../src/srcset';
 const ATTRIBUTES_TO_PROPAGATE = ['alt', 'title', 'referrerpolicy', 'aria-label',
   'aria-describedby', 'aria-labelledby'];
 
+
+const EXPERIMENTAL_ATTRIBUTES = ['srcset', 'src', 'sizes'];
+
+/**
+ * Regex for finding the first url in a srcset.
+ * @type {!RegExp}
+ */
+const FIRST_SRCSET_URL = /https?:\/\/\S+/;
```

Comment:
This won't handle relative urls which can happen on origin (cache rewrites all images). Probably safer to import `srcsetRegex` from `srcset.js` and grab the first item. (we still end up reducing the v0.js a lot anyway since most of code from `srcset.js` won't get imported after this is fully launched.

Reply:
(this regex was fine for preconnect case since that only matters for cache cases where img Urls are fully rewritten anyway)